.prettierignore

```diff
@@ -1,2 +1,5 @@
 CODEOWNERS
+packages/**/__generated__/**
+packages/apps/kadena-docs/src/data/menu.mjs
```

Comment:
why are these added just out of curiosity?

Reply:
Because with this PR we're formatting more code. The previous setup wasn't complete. And still isn't tbqh, but we'll get to that soon enough.